packages/fether-react/src/Send/TxForm/TxForm.js

```diff
@@ -473,12 +491,24 @@ class TxForm extends Component {
       // Verify that `gas + (eth amount if sending eth) <= ethBalance`
       if (
         this.estimatedTxFee(values)
-          .plus(token.address === 'ETH' ? toWei(values.amount) : 0)
+          .plus(
+            token.address === 'ETH' || token.address === 'ETC'
+              ? toWei(values.amount)
+              : 0
+          )
           .gt(toWei(values.ethBalance))
       ) {
-        return token.address !== 'ETH'
-          ? { amount: 'ETH balance too low to pay for gas' }
-          : { amount: "You don't have enough ETH balance" };
+        return token.address !== 'ETH' && token.address === 'ETC'
+          ? {
+            amount: `${
+              currentChainIdBN.valueOf() === '61' ? 'ETC' : 'ETH'
+            } balance too low to pay for gas`
+          }
+          : {
+            amount: `You don't have enough ${
+              currentChainIdBN.valueOf() === '61' ? 'ETC' : 'ETH'
```

Comment:
might be worth creating util functions somewhere
`isEthChainId(chainId: number | BigNumber) : boolean` (that accepts a number or a BN)
`chainIdToString(chainId: number | BigNumber) : string`

Reply:
great idea! i've addressed it in this commit b31e4336d74afa82abaaba20d632770252f15faa